### ISSUE TITLE
What is the threshold for deployment?

### ISSUE THREAD

**Issue by hernzzzzzzzz** [bitcoinclassic/bitcoinclassic#33]
Previous BIPs for bitcoin had Supermajority at 95% XT introduced 75% + short grace period. Such a low threshold opens up a 51% style attack where a lucky streak could occur... i.e. BC is running 40% of hash rate but clear 750 blocks... The BC chain would activate with only 40% on board essentially splitting the network 60/40 with Bitcoin Classic on the lower %

**Comment by dgmib** [comment]
You're ridiculously overestimating the odds of a "lucky streak" triggering a fork without a majority.

The probability of "winning" 750 out of 1000 consecutive blocks with 50% or less of the hash power is 4.3 x 10^-59.   **In layman's terms there a better chance of someone winning the USA "Power Ball" jackpot 8 times in a row.**

Or put another way... it will happen once every 400,000,000 billion billion billion billion billion years.

Even the odds of getting 750/1000 of the block with 70% of the hash power is virtually nil.  750/1000 blocks is more than enough to ensure a definitive super majority.

**Comment by jtoomim** [comment]
I would like to change the code to only test the forking condition once every difficulty adjustment instead of every single block. That addresses the luck issue a lot.

With a 750/1000 threshold, there is a significant chance of activating with only 70% of the actual hashrate within a month or two. Support below that has very rapidly diminishing probabilities.

There are three dangers to using an excessively high threshold:
1. It paints a target on the back of small miners. Bitcoin is supposed to be able to make censorship-resistant decisions. If 95% support is required to make this decision, then someone could harass or blackmail a miner and convince them to not support the fork, thereby blocking the ability to make a decision. Coersion could even be in the form of DDoS attacks, like was seen with Slush's pool on XT. The higher the threshold, the more effective these types of tactics are. This is not simply an academic concern: Many of the Bitcoin Core developers have received death threats in the last year. One committer to Core released his commit access and dropped out of public view, likely as a result. The use of dark net marketplaces with Bitcoin payments for illicit services to carry out these attacks makes them even more plausible.
2. A 51% majority could enforce any other majority if it wanted to. In magical game theory land, the only threshold of miner support that is justifiable is 51%. Everything above that falls into the realm of ensuring smooth deployment.
3. The higher the threshold, the greater the power of a minority. With a 75% threshold, "Nay" votes are 3x as potent as "Yea" votes. I think that's more than enough. At 95%, "Nay" votes would be 19x as potent. I think that's crazy.

Some earlier comments I made on reddit:

https://www.reddit.com/r/Bitcoin_Classic/comments/40wnre/on_activation_thresholds/

Let's take the discussion there, please, as it is a more appropriate forum for politics.

**Comment by dskloet** [comment]
There's a good chance of activating at 72% but definitely not at 70% the odds are quickly diminishing.
https://www.reddit.com/r/btc/comments/418xzh/the_odds_of_forking_with_75_hash_power/

Changing the activation code will take time. I wouldn't do it unless the miners demand it. But it would be easier to just change the activation threshold to 780/1000 blocks. Or 1500/2000 blocks or something.

**Comment by hernzzzzzzzz** [comment]
I wouldn't call a HF activation threshold political but thanks for addressing my concern.

**Comment by jtoomim** [comment]
Since this code is now deployed, I think that it doesn't make sense to leave this issue open any longer.
